ansible/meta.yml

```diff
@@ -0,0 +1,37 @@
+# This playbook gathers meta data facts for SAP/HANA checker.
+
+- hosts: localhost
+
+  tasks:
+    - name: include load_facts
```

Comment:
I don't think we need to load the facts here.
Let's remove it if it is not completely needed